model/ParsedInterface.java

```diff
@@ -3,13 +3,11 @@
 import java.io.File;
 import java.io.IOException;
 
-public interface ParserInterface {
+public interface ParsedInterface {
 
-  void setFile(File file);
+  File giveFile();
 
-  File getFile();
-
-  String getContent() throws IOException;
+  String giveContent() throws IOException;
```

Comment:
@undsett why not just `content`? The main reason to avoid compound names - is that the class provides context, like `parsed.content()` - looks nice vs imperative(hence bad since we do not "trust" `parsed` instance) `parsed.giveContent()`

Reply:
sounds reasonable, but saveContent is still fine, right?